org.tahoma2d.Tahoma2D.yaml

```diff
@@ -28,166 +24,230 @@ modules:
+      - -DOPENCV_ENABLE_NONFREE=ON
+      - -DOPENCV_GENERATE_PKGCONFIG=ON
+      - -DPROTOBUF_UPDATE_FILES=ON
+      - -DWITH_1394=OFF
+      - -DWITH_CUDA=OFF
+      - -DWITH_EIGEN=ON
+      - -DWITH_FFMPEG=ON
+      - -DWITH_GPHOTO2=OFF
+      - -DWITH_GSTREAMER=ON
+      - -DWITH_JASPER=OFF
+      - -DWITH_OPENEXR=ON
+      - -DWITH_OPENGL=OFF
+      - -DWITH_QT=OFF
+      - -DWITH_TBB=ON
+      - -DWITH_VTK=ON
+      - -DBUILD_opencv_python2=OFF
+      - -DBUILD_opencv_python3=ON
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/opencv/opencv/archive/4.5.1.tar.gz
+        sha256: e27fe5b168918ab60d58d7ace2bd82dd14a4d0bd1d3ae182952c2113f5637513
 
-    - name: opencv
-      buildsystem: cmake-ninja
-      builddir: true
-      config-opts:
-        - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-        - -DOPENCV_GENERATE_PKGCONFIG=ON
-        - -DWITH_OPENCL=ON
-        - -DWITH_OPENGL=ON
-        - -DWITH_TBB=ON
-        - -DWITH_VULKAN=ON
-        - -DBUILD_WITH_DEBUG_INFO=OFF
-        - -DBUILD_TESTS=OFF
-        - -DBUILD_PERF_TESTS=OFF
-        - -DBUILD_EXAMPLES=OFF
-        - -DINSTALL_C_EXAMPELS=OFF
-        - -DINSTALL_PYTHON_EXAMPLES=OFF
-        - -DWITH_1394=OFF
-        - -DWITH_PROTOBUF=ON
-        - -DBUILD_LIST=alib3d,core,dnn,features2d,flann,highgui,imgcodecs,imgproc,java_bindings_generator,ml,objdetect,photo,python_bindings_generator,python_tests,stitching,video,videoio
-      cleanup:
-        - /bin
-      sources:
-        - type: archive
-          url: https://github.com/opencv/opencv/archive/4.2.0.tar.gz
-          sha256: 9ccb2192d7e8c03c58fee07051364d94ed7599363f3b0dce1c5e6cc11c1bb0ec
-        - type: archive
-          url: https://github.com/opencv/opencv_contrib/archive/4.2.0.tar.gz
-          sha256: 8a6b5661611d89baa59a26eb7ccf4abb3e55d73f99bb52d8f7c32265c8a43020
-    
-    - name: tahoma2d
-      buildsystem: cmake
-      build-commands:
-        #- install -Dm 644 -t /app/share/applications com.tahoma2d.Tahoma2D.desktop
-        #- install -Dm 644 -t /app/share/icons/hicolor/256x256/apps com.tahoma2d.Tahoma2D.png
-      builddir: true
-      subdir: toonz/sources
-      config-opts:
-        - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-        - -DWITH_TRANSLATION=OFF
-        - -DBOOST_ROOT=/app/include/boost
-      sources:
-        - type: archive
-          url: https://github.com/tahoma2d/tahoma2d/releases/download/v1.4/Tahoma2D-linux.tar.gz
-          sha256: 39616a1157f1fcf250914d737128c60a1e202f93fd91cbb962e4b1b9a07d7c22
-        - type: shell
-          commands:
-            - cd thirdparty/tiff-2.0 &&
-              ./configure --with-pic --disable-jbig &&
-              make -j $FLATPAK_BUILDER_N_JOBS
-        - type: file
-          path: com.tahoma2d.Tahoma2D.png
-        - type: file
-          path: com.tahoma2d.Tahoma2.metainfo.xml
-        - type: file
-          path: com.tahoma2d.Tahoma2D.desktop
+  - name: libgphoto2
+    buildsystem: simple
```

Comment:
this should NOT be buidsystem: simple

Reply:
Care to make a suggestion?

Reply:
like every other flatpak that build it. We have support for autotools in flatpak-builder from day 1.

Reply:
Do you have an example of this from another flathub project I can model?

Sorry, this is all new to me and I'm trying to figure this all out.  At it were, I actually had to build the boost library (OT wasn't really building it) and I was lucky to find a few flathub projects that did it already so I just modeled it after those.

Reply:
Nevermind. I found an example